sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  - added support for copying `image_mem_handle` between devices via
+    `ext_oneapi_copy`. intel/llvm#12449
+  - added `fetch_image` overload which accepts sampled image and coordinates.
+    intel/llvm#12447
+  - added type hint template argument to `fetch_cubemap` and `sample_cubemap`
+    APIs. intel/llvm#13742
+
+### SYCLcompat
+
+- Added non-`const` `image2d_max` and `image3d_max` getters. intel/llvm#14138
+- Introduced versioning scheme for the library. intel/llvm#14457
+- Enhanced  masked shuffle functions `select_from_sub_group`,
+  `shift_sub_group_left`, `shift_sub_group_right` and `permute_sub_group_by_xor`
+  to support CUDA devices. intel/llvm#13363
+- Restricted `memory_order` argument of `atomic_ref` passed to
+  `experimental::nd_range_barrier` to match supported on a device.
+  intel/llvm#12974 intel/llvm#13641
+
+## Bug Fixes
+
+### SYCL Compiler
+
+- Fixed that using `-fsycl-link-targets` flag would inadvertently trigger some
+  additional device code linking steps. intel/llvm#13004
+- Fixed a bug that when AOT-compiling for Intel GPUs the compiler would pass
+  some PVC-specific flags even if target device is not a PVC. intel/llvm#13794
+- Fixed a bug with incorrect file extensions being emitted in AOT compilation
+  when `--save-temps` is used. intel/llvm#14214
+- Made `-fsycl-add-default-spec-consts-image` available with `clang-cl`. intel/llvm#13168
+- Fixed an issue where performing separate compilation and linking with
+  `-fsycl-link` would result in "number of output files and targets should match
+  in unbundling mode" error emitted by the compiler during link step. intel/llvm#13002
+- Fixed a bug in address sanitizer which may lead to crashes when an application
+  is launched on OpenCL CPU device. intel/llvm#13262
+- Fixed a bug where calling certain built-in math functions that accepts
+  pointers (like `fract`, `frexp`, `modf`, etc.) and passing pointer in the
+  generic address space there would not compile for AMD devices.
+  intel/llvm#13015 intel/llvm#13092 intel/llvm#13361 intel/llvm#13792
+  intel/llvm#13546
+- Fixed a bug where compiling a program that contains kernels with different
+  `reqd_work_group_size` attributes attached to them using
+  `-fsycl-device-code-split=none` would result in an exception being thrown at
+  runtime about mismatching work-group size. intel/llvm#13523
+- Fixed a bug where compiling a kernel that is annotated with
+  `reqd_work_group_size` attribute that has less than 3 arguments for HIP
+  target caused compiler to crash. intel/llvm#13600
+- Fixed a bug with `shift_group_[right|left]`, `permute_by_xor` and
+  `select_from_group` algorithms would return invalid values if used with
+  `half` data type on AMD devices. intel/llvm#13016
+
+### SYCL Library
+
+- Fixed a situation when querying
+  `sycl::ext::oneapi::experimental::info::device` could result in exception
+  being thrown instead of empty vector being returned. intel/llvm#13968
```

Comment:
```suggestion
- Fixed a situation when querying
  `sycl::ext::oneapi::experimental::info::device` could result in an exception
  being thrown instead of an empty vector being returned. intel/llvm#13968
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706